packages/tao/src/commands/migrate.ts

```diff
@@ -459,6 +460,8 @@ function createFetcher(packageManager: PackageManager) {
           version: resolvedVersion,
         };
       }
+      console.log('cleaning up', dir);
+      rmdirSync(dir, { recursive: true });
```

Comment:
I suggest using `removeSync` from 'fs-extra' because the `recursive` option of `rmdirSync` is deprecated see `https://nodejs.org/api/fs.html#fs_fs_rmdirsync_path_options`.

Reply:
Good idea. I'll use `rmSync` instead as recommended there

Reply:
@FrozenPandaz rmSync is not safe to use because it is not supported in Node 12. I think #5896 is related...